content/blog/2020-10-12-october-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,162 @@
+---
+title: October '20 Heartbeat
```

Comment:
so, does it include September's content? or October's .... I'm lost a bit here :)

Reply:
Should include anything from the last heartbeat (https://dvc.org/blog/september-20-dvc-heartbeat) to now- there are some October events in there, like PyCon India because it was right on the edge. But should mostly be content from the last 4 weeks, as usual. Does that make sense?

Reply:
so, it's roughly mid Sep - mid Oct - makes sense ... I see at least 3-4 events/links in October.

overall, `(last Heartbeat date, new Heartbeat publish date]` makes total sense to me, thanks.